pontoon/sync/formats/json_keyvalue.py

```diff
@@ -0,0 +1,158 @@
+"""
+Parser for the .json translation format as used by the WebExtensions API:
+https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Internationalization
+
+See also:
+https://www.chromium.org/developers/design-documents/extensions/how-the-extension-system-works/i18n
```

Comment:
This is a remnant from the original `json_extensions.py` that should be updated, yes?

Reply:
Yes indeed